Pythonwin/pywin/mfc/dialog.py

```diff
@@ -33,10 +33,10 @@ def __init__(self, id, dllid=None):
         dllid may be None, a dll object, or a string with a dll name"""
         # must take a reference to the DLL until InitDialog.
         self.dll = dllFromDll(dllid)
-        if type(id) == type([]):  # a template
```

Comment:
isn't this (and the similar change below) wrong?

Reply:
I fixed up the PR and removed the changes that shouldn't have been in there. Ready to review :)